### PR TITLE
Add buttons to add Emerald to MetaMask

### PR DESCRIPTION
**Pull request by lukaw3d**
https://deploy-preview-80--trusting-archimedes-14c863.netlify.app/general/developer-resources/emerald-paratime/#web3-gateway
image

### CODE REVIEW

**Review by gparty — APPROVED** [review]
Great new feature, working great in Metamask and Brave (built in) Wallet.

File: docs/general/developer-resources/emerald-paratime/README.mdx [hunk 1]
```diff
@@ -36,6 +36,30 @@ To get started building on our Emerald ParaTime, you can use our public Web3 gat
...
+            decimals: 18,
+          },
+          rpcUrls: ['https://emerald.oasis.dev', 'wss://emerald.oasis.dev/ws'],
+          blockExplorerUrls: ['https://explorer.emerald.oasis.dev'],
+        },
+      ],
+    })
+  }
+>
+  Add Emerald Mainnet to MetaMask
```
Comment (lukaw3d) [inline docs/general/developer-resources/emerald-paratime/README.mdx]: @gparty Oh nice. In that case:
```suggestion
  Add Emerald Mainnet to MetaMask or Brave Wallet
```

**Review by gparty — APPROVED** [review]
GJ

File: docs/general/developer-resources/emerald-paratime/README.mdx [hunk 2]
```diff
@@ -36,6 +36,30 @@ To get started building on our Emerald ParaTime, you can use our public Web3 gat
...
+            decimals: 18,
+          },
+          rpcUrls: ['https://emerald.oasis.dev', 'wss://emerald.oasis.dev/ws'],
+          blockExplorerUrls: ['https://explorer.emerald.oasis.dev'],
+        },
+      ],
+    })
+  }
+>
+  Add Emerald Mainnet to MetaMask or Brave Wallet
```
Comment (matevz) [inline docs/general/developer-resources/emerald-paratime/README.mdx]: I would put it more explicit. Same for testnet.

```suggestion
  Click here to add Emerald Mainnet to MetaMask or Brave Wallet
```

**Review by matevz — CHANGES_REQUESTED** [review]
I didn't notice the buttons immediately because of a very soft border. Could you maybe add a shadow similar to DocCards. Or maybe show it as `<a href="">`

File: docs/general/developer-resources/emerald-paratime/README.mdx (hunk 2 above)
Comment (matevz) [reply docs/general/developer-resources/emerald-paratime/README.mdx]: Even better, could you move the button to the top just below the Mainnet subtitle and change the label to

 ```suggestion
  Click here to add Emerald Mainnet to MetaMask or Brave Wallet automatically
```

So folks see the button before they start reading the actual chain ids, endpoints etc. (and already on the way to filling metamask fields).

File: docs/general/developer-resources/emerald-paratime/README.mdx [hunk 3]
```diff
@@ -36,6 +36,30 @@ To get started building on our Emerald ParaTime, you can use our public Web3 gat
   * Decimal: 42262
 * Block explorer: [https://explorer.emerald.oasis.dev](https://explorer.emerald.oasis.dev)
 
+<button
+  class="button button--secondary button--outline"
```
Comment (lukaw3d) [inline docs/general/developer-resources/emerald-paratime/README.mdx]: @matevz to make button more noticeable, the easiest is to switch to one of these variants:

- current, button--secondary button--outline
current_outline-secondary

- button--outline
outline

- button--secondary
secondary

- button--primary
primary

Comment (lukaw3d) [reply docs/general/developer-resources/emerald-paratime/README.mdx]: oh wait, button--outline doesn't work well when hovered

File: docs/general/developer-resources/emerald-paratime/README.mdx (hunk 2 above)
Comment (lukaw3d) [reply docs/general/developer-resources/emerald-paratime/README.mdx]: New preview

image

Comment (lukaw3d) [reply docs/general/developer-resources/emerald-paratime/README.mdx]: How about: 
image

File: docs/general/developer-resources/emerald-paratime/README.mdx [hunk 4]
```diff
@@ -29,6 +29,30 @@ To get started building on our Emerald ParaTime, you can use our public Web3 gat
...
+            decimals: 18,
+          },
+          rpcUrls: ['https://emerald.oasis.dev', 'wss://emerald.oasis.dev/ws'],
+          blockExplorerUrls: ['https://explorer.emerald.oasis.dev'],
+        },
+      ],
+    })
+  }
+>
+  Click here to add Emerald Mainnet to MetaMask or Brave Wallet automatically
```
Comment (matevz) [inline docs/general/developer-resources/emerald-paratime/README.mdx]:
```suggestion
  Click here to register Emerald Mainnet to your MetaMask or Brave Wallet
```

File: docs/general/developer-resources/emerald-paratime/README.mdx [hunk 5]
```diff
@@ -38,6 +62,30 @@ To get started building on our Emerald ParaTime, you can use our public Web3 gat
...
+            decimals: 18,
+          },
+          rpcUrls: ['https://testnet.emerald.oasis.dev/', 'wss://testnet.emerald.oasis.dev/ws'],
+          blockExplorerUrls: ['https://testnet.explorer.emerald.oasis.dev'],
+        },
+      ],
+    })
+  }
+>
+  Click here to add Emerald Testnet to MetaMask or Brave Wallet automatically
```
Comment (matevz) [inline docs/general/developer-resources/emerald-paratime/README.mdx]:
```suggestion
  Click here to register Emerald Testnet to your MetaMask or Brave Wallet
```

**Review by matevz — APPROVED** [review]
The blue style looks great. IMO you should move the button back at the end of the subsection now with slightly changed labels:
image

**Pull request merged**